wagtail/admin/tests/test_edit_handlers.py

```diff
@@ -460,6 +460,46 @@ def test_render_form_content(self):
         # in the panel definition
         self.assertNotIn("signup_link", result)
 
+    def test_tabs_permissions(self):
+        # self.assertIn("Abergavenny sheepdog trials</textarea>", "result")
+        # test that two tabs show when the current has permission to see both
+        # check that one tab shows when the current user has permission to only single one
+        event_page_tabbed_interface = TabbedInterface(
```

Comment:
now that you can use the `self.event_page_tabbed_interface` you should not need to build a custom one here.